docs/quickstart.ipynb

```diff
@@ -4,20 +4,18 @@
    "cell_type": "markdown",
```

Comment:
<p>Link to the docs regarding progress bars and convergence plots. Also link to the docs regarding the log level.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Done.